tool/tbot/config/config.go

```diff
@@ -103,7 +103,7 @@ type OnboardingConfig struct {
 	Token string `yaml:"token"`
 
 	// CAPath is an optional path to a CA certificate.
-	CAPath string
+	CAPath string `yaml:"ca_pin"`
```

Comment:
Looks like a name mismatch?

```suggestion
	CAPath string `yaml:"ca_path"`
```